library/Vanilla/FeatureFlagHelper.php

```diff
@@ -19,22 +18,40 @@ class FeatureFlagHelper {
      */
     public static function featureEnabled(string $feature): bool {
         // We're going to enforce the root "Feature" namespace.
-        $configValue = c("Feature.{$feature}.Enabled");
+        $configValue = Gdn::config("Feature.{$feature}.Enabled");
```

Comment:
Especially if we're going to be using these feature flags more often, we should update this config check to use `StaticCacheConfigTrait`.

Reply:
I don't think feature flags are necessarily going to get a lot of mileage out of static config caching, since they probably won't be checked multiple times in the same request (or at least enough to see a real return here). However, I don't think it'll hurt anything either, so it's been implemented.